src/user/using-the-aggregate-data-entry-app.md

```diff
@@ -0,0 +1,283 @@
+
+1.  Click on "Add comment"
+
+    ![](resources/images/data_entry_beta/add_comment.png)
+
+3.  Enter the comment
+
+4.  Click **Save**.
+
+    ![](resources/images/data_entry_beta/save_comment.png)
+
+## Create minimum maximum value range manually { #aggregate_data_entry_app.change_min_max_range_manually } 
+
+1.  In the **Data Entry** app (beta), open a data entry form.
+
+1.  Open the data value's details for which you want to set the minimum maximum
+    value range.
+
+1.  Click on "Add limits"
+
+    ![](resources/images/data_entry_beta/add_limits.png)
+
+3.  Enter **Min limit** and **Max limit**.
+
+4.  Click **Save**.
+
+    ![](resources/images/data_entry_beta/save_new_limits.png)
+    
+    If values don't fall within the new value range the next time you enter
+    data, the data entry cell will appear with a red background.
+
+## Display a data value's history { #aggregate_data_entry_app.display_data_value_history } 
+
+You can display the last 12 values registered for a field.
+
+1.  Open the **Data Entry** app (beta).
+
+1.  Open an existing data entry form.
+
+1.  Open the data value's details
+
+4.  Click **History**.
+
+![](resources/images/data_entry_beta/history.png)
+
+## Display a data value's audit log { #aggregate_data_entry_app.display_data_value_audit_log } 
+
+The audit log allows you to view other data values which have been entered
+prior to the current value. The audit trail also shows when the data value was
+altered and which user who made the changes.
+
+1.  Open the **Data Entry** app (beta).
+
+1.  Open an existing data entry form.
+
+1.  Open the data value's details
+
+4.  Click **Audit log**.
+
+![](resources/images/data_entry_beta/audit_log.png)
+
+## Enter data offline { #aggregate_data_entry_app.enter_data_offline } 
+
+The **Data Entry** app (beta) works even if you don't have a stable Internet
+connection during data entry. When you don't have an internet
+connection, the data you enter is saved to your local computer. When the
+Internet connection is back, the app will push the data to the server.
+
+> **Note**
+> 
+> To use this functionality, you must login to the server while you've
+> an Internet connection.
+
+-   When you're connected to the Internet, the app displays this message
+    at the top of the data entry form:
+    
+    ![](resources/images/data_entry_beta/data_entry_online1.png)
+
+-   If your Internet connection breaks during data entry, the app
+    detects it and displays this message:
+    
+    ![](resources/images/data_entry_beta/data_entry_offline1.png)
+    
+    Now your data will be stored locally. You can continue to enter data
+    as normal.
+
+-   Once you have entered data and the app detects that the Internet connection
+    is back, the data will be sent automatically:
+
+## Multi-organisation unit data entry { #aggregate_data_entry_app.data_entry_multiple_organisation_units } 
+
+The **Data Entry** app (beta) currently does not support multi-organisation
+units!
+
```

Comment:
Do we need to say this? Why not remove altogether. 

```suggestion
```

Reply:
The reason I included it is: If a user is wondering how this works in the new app and don't find any information about it, they could be confused.

I can remove it if you think we shouldn't put it in there

Reply:
To be honest I wasn't familiar with this feature, but I see it is in the old app. 
In that case, I agree it may make sense to mention it as you have done.